packages/app-staking/src/index.css

```diff
@@ -18,6 +18,7 @@
 
   .ui--AddressMini {
     display: block;
+    text-align: right;
```

Comment:
For short names, align icon to the very right (it goes left by default when the balance determines the width)